app/services/valid_test_data_generators/applications_populater.rb

```diff
@@ -44,6 +45,13 @@ def create_participants!
       end
     end
 
+    def prepare_cohort!
+      cohort.tap do |c|
+        c.funding_cap = cohort.start_year >= 2024
```

Comment:
Can we use something like `Cohort.current.start_year` here instead of hard coding?